LayoutTests/imported/w3c/web-platform-tests/css/cssom/serialize-values.html

```diff
@@ -262,10 +262,11 @@
       }],
       ['background-position', {
         'values': [[[percentage, negative_percentage, length, negative_length,
-                     'left', 'center', 'right'],
+                     'left', 'right'],
                     [percentage, negative_percentage, length, negative_length,
-                     'top', 'center', 'bottom']],
+                     'top', 'bottom']],
                    'inherit'],
+        //FIXME testing cases involving center requires understanding when it can be omitted
```

Comment:
I'm afraid nobody will address this, so I think you should do it now.

Reply:
There are many, many cases that this test doesn’t currently cover. Many properties just mentioned and commented out. To me, this one didn’t really stick out. Addressing this is a significant project because this test doesn’t cover any cases of omitting values from shorthands at all and is written in a very simple way. I will do it, but I was worried about making this large change a gigantic project. I could do all the WPT fixes separately first without any changes to WebKit, I guess, then this would be more manageable.